src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_5_horizontal_layout.html.twig

```diff
@@ -0,0 +1,130 @@
+            {%- if expanded is not defined or not expanded -%}
+                {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' col-form-label')|trim}) -%}
+            {%- endif -%}
+            {%- set label_attr = label_attr|merge({class: (label_attr.class|default('') ~ ' ' ~ block('form_label_class'))|trim}) -%}
+        {%- endif -%}
+        {{- parent() -}}
+    {%- endif -%}
+{%- endblock form_label %}
+
+{% block form_label_class -%}
+    col-sm-2
+{%- endblock form_label_class %}
+
+{# Rows #}
+
+{% block form_row -%}
+    {%- if expanded is defined and expanded -%}
+        {{ block('fieldset_form_row') }}
+    {%- else -%}
+        {%- set widget_attr = {} -%}
+        {%- if help is not empty -%}
+            {%- set widget_attr = {attr: {'aria-describedby': id ~"_help"}} -%}
+        {%- endif -%}
+        {%- set row_class = row_class|default(row_attr.class|default('mb-3')) -%}
+        {%- set is_form_floating = is_form_floating|default('form-floating' in row_class) -%}
+        {%- set is_input_group = is_input_group|default('input-group' in row_class) -%}
+        {#- Remove behavior class from the main container -#}
+        {%- set row_class = row_class|replace({'form-floating': '', 'input-group': ''}) -%}
+        <div{% with {attr: row_attr|merge({class: (row_class ~ ' row' ~ ((not compound or force_error|default(false)) and not valid ? ' is-invalid'))|trim})} %}{{ block('attributes') }}{% endwith %}>
+            {%- if is_form_floating or is_input_group -%}
+                <div class="{{ block('form_label_class') }}"></div>
+                <div class="{{ block('form_group_class') }}">
+                    {%- if is_form_floating -%}
+                        <div class="form-floating">
+                            {{- form_widget(form, widget_attr) -}}
+                            {{- form_label(form) -}}
+                        </div>
+                    {%- elseif is_input_group -%}
+                        <div class="input-group">
+                            {{- form_label(form) -}}
+                            {{- form_widget(form, widget_attr) -}}
+                            {#- Hack to properly display help with input group -#}
+                            {{- form_help(form) -}}
+                        </div>
+                    {%- endif -%}
+                    {%- if not is_input_group -%}
+                        {{- form_help(form) -}}
+                    {%- endif -%}
+                    {{- form_errors(form) -}}
+                </div>
+            {%- else -%}
+                {{- form_label(form) -}}
+                <div class="{{ block('form_group_class') }}">
+                    {{- form_widget(form, widget_attr) -}}
+                    {{- form_help(form) -}}
+                    {{- form_errors(form) -}}
+                </div>
+            {%- endif -%}
+        {##}</div>
```

Comment:
`{##}` what are those for?

Reply:
To control whitespace when they need to be removed. They go with the `-%}` above

Reply:
this one is unnecessary. `-%}` already removes those whitespace.

A comment is useful when you use `{#- #}` to apply whitespace trimming in a place where there is no Twig markup (the comment is a way to add such a markup that can hold the trimming behavior, but that's not the case here) or to end the whitespace trimming (allowing to keep a non-trimmed space after the comment)